g1ant-language/commands/timeout.reset.md

```diff
@@ -0,0 +1,22 @@
+# timeout.reset
+
+h1. KOMENDA NIE DZIALA!!!!!
```

Comment:
let's remove this polish comment shall we :p

Reply:
as above: all files need reviewing, but not now